third_party/amd/include/Dialect/TritonAMDGPU/IR/TritonAMDGPUOps.td

```diff
@@ -21,15 +21,90 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-
 #ifndef TRITON_AMDGPU_OPS
 #define TRITON_AMDGPU_OPS
 
 include "mlir/IR/OpBase.td"
 include "mlir/IR/EnumAttr.td"
 include "mlir/Dialect/LLVMIR/LLVMOpBase.td"
 include "mlir/Interfaces/InferTypeOpInterface.td"
+include "mlir/Interfaces/SideEffectInterfaces.td" // Pure
+include "mlir/Interfaces/ViewLikeInterface.td" // OffsetSizeAndStrideOpInterface
 include "TritonAMDGPUDialect.td"
 include "TritonAMDGPUAttrDefs.td"
 
+class TritonAMDGPU_Op<string mnemonic, list<Trait> traits = []>
+    : Op<TritonAMDGPU_Dialect, mnemonic, traits>;
+
+//===----------------------------------------------------------------------===//
+// ViewSliceOp
+//===----------------------------------------------------------------------===//
+
+def TritonAMDGPU_ViewSliceOp
+    : TritonAMDGPU_Op<"view_slice", [AttrSizedOperandSegments,
+                                     OffsetSizeAndStrideOpInterface, Pure]> {
+  let summary = "view slice operation";
+  let description = [{
+    The "view_slice" operation enables "viewing" a slice of a tensor in
```

Comment:
Nit: no need to quote "view"--view is sort of a common concept well known in languages/compilers already.